locator/tablets.hh

```diff
@@ -157,6 +157,7 @@ enum class tablet_transition_stage {
     write_both_read_new,
     use_new,
     cleanup,
+    cleanup_target,
```

Comment:
Perhaps we need more explicit names: cleanup_source, cleanup_target_before_revert.